test/integration/export-all-in-page-with-custom-page-extensions/test/index.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env jest */
+import { File, nextBuild } from 'next-test-utils'
+import { join } from 'path'
+
+const appDir = join(__dirname, '..')
+
+describe('Re-export all exports from page is disallowed for custom pageExtensions', () => {
+  it('shows error when a page re-export all exports', async () => {
+    const { code, stderr } = await nextBuild(appDir, undefined, {
+      stderr: true,
+    })
+    expect(code).toBe(1)
+    expect(stderr).toMatch(/\/export-all-in-page/)
+
+    expect(stderr.split('\n\n')[1]).toMatchInlineSnapshot(`
+      "./pages/contact.page.js:3:1
+      Syntax error: Using \`export * from '...'\` in a page is disallowed. Please use \`export { default } from '...'\` instead.
+      Read more: https://nextjs.org/docs/messages/export-all-in-page"
+    `)
```

Comment:
This might need an update when the babel config is removed and this run uses SWC. https://github.com/vercel/next.js/pull/32426/files#r809646278

Reply:
Sure. will try to resolve these today or tomorrow. 
btw I have added test cases for both babel and swc https://github.com/vercel/next.js/pull/32426/files#diff-1eca3e23fa5d69b437524d88ae4b58e5485170dd32981e90b802dbe5a7469313R64
so I think I don't have to update this or delete babel config file